src/@next/Tag/Tag.tsx

```diff
@@ -17,5 +32,39 @@ export const Tag = ({ children, ...props }: TagProps) => {
       children
     );
 
-  return <TagStyle {...props}>{content}</TagStyle>;
+  const removeButton = onRemove && (
+    <TagRemoveContainerStyle
+      role={'button'}
+      className={'icon-remove'}
+      tabIndex={0}
+      onMouseDown={() => setIsPressed(true)}
+      onBlur={() => setIsPressed(false)}
+      onMouseLeave={() => {
+        setIsPressed(false);
+        setIsHover(false);
+      }}
+      onMouseUp={() => setIsPressed(false)}
+      onMouseEnter={() => setIsHover(true)}
+    >
+      <Icon
+        name={'ri-close'}
```

Comment:
please remove `{` and `}` when you pass strings

Reply:
removed